### ISSUE TITLE
Assignment 19 -- Progress Report

### ISSUE THREAD

**Issue by dudung** [dudung/sk5010-01-2024-1#19]
Provide following key elements for you progress report as described in the [slide](https://osf.io/xc3d8).

Report that in the form of a table as follow

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | ..
1 | Data Source and Accessibility | ..
2 | Research Question or Focus | ..
3 | Methods for Visualization | ..
4 | Programming Language and Tools | ..
5 | Platforms and Technical Setup | ..
6 | Types of Output | ..
7 | Timeline and Next Steps | .. 

with Markdown code

```
No | Key Element | Description
:- | :- | :-
0 | Tentative Title | ..
1 | Data Source and Accessibility | ..
2 | Research Question or Focus | ..
3 | Methods for Visualization | ..
4 | Programming Language and Tools | ..
5 | Platforms and Technical Setup | ..
6 | Types of Output | ..
7 | Timeline and Next Steps | .. 
```
And put also your ID and name before the table.

Estimation time ~ 6 hours

**Comment by ilonajoan07** [comment]
No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualization of Loan Approval Dataset to Predict Financial Risk
1 | Data Source and Accessibility | Kaggle: https://www.kaggle.com/datasets/lorenzozoppelletto/financial-risk-for-loan-approval
2 | Research Question or Focus | What components from the dataset that have high/low correlation to each other? From the visualisations, what can be seen (trends, correlations)? Can a good prediction model be made to predict the approval of loan demand and the financial risk level of the individual?
3 | Methods for Visualization | Multiple Component Analysis, PCA, Plotting
4 | Programming Language and Tools | Python (Seaborn, scikit), Jupiter Notebook, matplotlib
5 | Platforms and Technical Setup | Kaggle, Excel, Python, Jupiter Notebook, Github, OSF
6 | Types of Output | Visualisations from the dataset, heatmap, correlation parameters, 2D/3D plots, graphs, scatter plots, codes (put together in a final report with journal format)
7 | Timeline and Next Steps | (table below)

Activity | Week 1 | Week 2 | Week 3 | Week 4 | Week 5
:- | :- | :- | :- | :- | :- 
Finding Topic | V | .. | .. | .. | ..
Dataset Search | V | .. | .. | .. | ..
Data Preparation | .. | V | .. | .. | ..
Visualizing and Analyzing | .. | .. | V | V | ..
Report Making | .. | .. | .. | .. | V

**Comment by azainiz** [comment]
Ahmad Zaini Zahrandika (20923302)

NOTE: Descriptions are written in Bahasa Indonesia

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi Kepadatan Penduduk di setiap Provinsi di Indonesia
1 | Data Source and Accessibility | <ul><li>Peta Indonesia: https://github.com/Vizzuality/growasia_calculator/tree/master/public </li><li>Kode program: https://www.kaggle.com/code/farizdarari/simple-map-visualization-using-geopandas </li><li>Jumlah penduduk menurut provinsi di Indonesia (ribu jiwa): https://sulut.bps.go.id/id/statistics-table/2/OTU4IzI=/jumlah-penduduk-menurut-provinsi-diindonesia.html </li><li>Luas daerah menurut provinsi di Indonesia: https://www.bps.go.id/id/statistics-table/3/VUZwV01tSlpPVlpsWlRKbmMxcFhhSGhEVjFoUFFUMDkjMw==/luas-daerah-dan-jumlah-pulau-menurut-provinsi.html </li></ul>
2 | Research Question or Focus | Membandingkan kepadatan penduduk atau *population density* di setiap provinsi di Indonesia melalui visualisasi, terutama di Provinsi DKI Jakarta yang diduga memiliki kepadatan penduduk tertinggi dibandingkan dengan provinsi lainnya.
3 | Methods for Visualization | <ul><li>Peta koroplet</li><li>Diagram batang</li></ul>
4 | Programming Language and Tools | <ul><li>Python, dengan *library* Pandas, Geopandas, dan Matplotlib. </li><li> Microsoft Excel, untuk menggabungkan data dari Badan Pusat Statistik Indonesia (BPS). </li></ul>
5 | Platforms and Technical Setup | <ul><li>Sistem operasi Windows 10 atau 11. </li><li>JupyterLab untuk menulis kode program Python dan menghasilkan file berupa __.ipynb__. </li><li> Github dan/atau Medium untuk menunjukkan output RBL. </li></ul>
6 | Types of Output | Kode program yang akan diunggah di Github, dan/atau artikel (*story*) di Medium.
7 | Timeline and Next Steps | <ol><li>Akan ditambahkan. </li><li> Ditunggu. </li></ol>

**Comment by azainiz** [comment]
Ahmad Zaini Zahrandika (20923302)

NOTE: Descriptions are written in Bahasa Indonesia

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi Kepadatan Penduduk di Setiap Provinsi di Indonesia
1 | Data Source and Accessibility | <ul><li>Kode program: https://www.kaggle.com/code/farizdarari/simple-map-visualization-using-geopandas </li><li>Peta Indonesia: https://github.com/Vizzuality/growasia_calculator/tree/master/public </li><li>Jumlah penduduk menurut provinsi di Indonesia (ribu jiwa): https://sulut.bps.go.id/id/statistics-table/2/OTU4IzI=/jumlah-penduduk-menurut-provinsi-diindonesia.html </li><li>Luas daerah menurut provinsi di Indonesia: https://www.bps.go.id/id/statistics-table/3/VUZwV01tSlpPVlpsWlRKbmMxcFhhSGhEVjFoUFFUMDkjMw==/luas-daerah-dan-jumlah-pulau-menurut-provinsi.html </li></ul>
2 | Research Question or Focus | Membandingkan kepadatan penduduk atau *population density* di setiap provinsi di Indonesia melalui visualisasi, terutama di Provinsi DKI Jakarta yang diduga memiliki kepadatan penduduk tertinggi dibandingkan dengan provinsi lainnya.
3 | Methods for Visualization | <ul><li>Peta koroplet</li><li>Diagram batang</li></ul>
4 | Programming Language and Tools | <ul><li>Python, dengan *library* Pandas, Geopandas, dan Matplotlib. </li><li> Microsoft Excel, untuk menggabungkan data dari Badan Pusat Statistik Indonesia (BPS). </li></ul>
5 | Platforms and Technical Setup | <ul><li>Sistem operasi Windows 10 atau 11. </li><li>JupyterLab untuk menulis kode program Python dan menghasilkan file berupa __.ipynb__. </li><li> Github dan/atau Medium untuk menunjukkan output RBL. </li></ul>
6 | Types of Output | Kode program yang akan diunggah di Github, dan/atau artikel (*story*) di Medium.
7 | Timeline and Next Steps | (dicantumkan di bawah tabel ini)

Lini Masa RBL (dengan minggu ke-1 pada tanggal 6-13 November 2024)

Kegiatan | Minggu ke-1 | Minggu ke-2 | Minggu ke-3 | Minggu ke-4 | Minggu ke-5
:- | :- | :- | :- | :- | :-
Mencari topik dan data | ✓ | .. | .. | .. | ..
Pengolahan dan analisis data | .. | ✓ | ✓ | .. | ..
Penyusunan laporan | .. | .. | .. | ✓ | ..
Publikasi | .. | .. | .. | .. | ✓

**Comment by MarcoSutisna** [comment]
Marco Sutisna Putra
ID: 20924001

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Implementasi Machine Learning untuk Deteksi Gambar Kemasan Makanan untuk Sistem Rekomendasi kandungan nutrisi harian
1 | Data Source and Accessibility | Kaggle: https://www.kaggle.com/datasets/parjunwoo/fooddatasert
2 | Research Question or Focus | Bagaimana penerapan machine learning untuk mendeteksi gambar? Bagaimana cara memastikan kebutuhan nutrisi harian setiap orang? Berapa efisiensi metode deteksi gambar yang digunakan?
3 | Methods for Visualization | Deteksi Gambar: CNN, Sistem Rekomendasi: (masih research), Menghubungkan data gambar dengan data nutrisi: Basis Data
4 | Programming Language and Tools | Python, MySQL (SQLYog)
5 | Platforms and Technical Setup | Basis data: SQLYog, Program: Google Collab dan VSCode
6 | Types of Output | Jurnal atau artikel medium
7 | Timeline and Next Steps | (dicantumkan pada tabel berikut)


| Kegiatan | Week 1 | Week 2 | Week 4 | Week 4 | Week 5 |
|---|---|---|---|---|---|
| Data Preprocessing | v | - | - | - | - |
| Penerapan Machine Learning | - | v | v | - | - |
| Penyusunan Basis Data | - | - | v | v | - |
| Pembuatan Sistem Rekomendasi | - | - | - | v | - |
| Penyusunan Jurnal | - | v | v | v | v |

**Comment by gerendc** [comment]
Gerend Christopher (20924009)

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi Tingkat Penyelesaian Pendidikan Setiap Provinsi di Indonesia
1 | Data Source and Accessibility | Tingkat Penyelesaian Pendidikan Setiap Provinsi di Indonesia (https://www.bps.go.id/id/statistics-table/2/MTk4MCMy/tingkat-penyelesaian-pendidikan-menurut-jenjang-pendidikan-dan-provinsi.html)
2 | Research Question or Focus | Menentukan penurunan tingkat penyelesaian pendidikan tiap provinsi dari tingkat SD hingga SMA.
3 | Methods for Visualization | - Choropleth map <br> - Bar Chart
4 | Programming Language and Tools | - Tableau Public (main) / Python (alternative) <br> - Microsoft Excel (pengolahan/processing data)
5 | Platforms and Technical Setup | - MacOS or Windows operating system <br> - Medium untuk menunjukkan output
6 | Types of Output | - JPG/PNG image file of choropleth map and bar chart <br> - Artikel Medium
7 | Timeline and Next Steps | (dicantumkan pada tabel di bawah)

Lini Masa RBL (Minggu ke-1 dimulai dari tanggal 6 November 2024)
Kegiatan | Minggu ke-1 | Minggu ke-2 | Minggu ke-3 | Minggu ke-4 | Minggu ke-5
:- | :- | :- | :- | :- | :-
Topik dan Data | ✓ | ✓ | ✓ |  |  
Pengolahan Data |  | ✓ | ✓ |  |  
Visualisasi |  |  | ✓ | ✓ |  
Penyusunan laporan |  |  |  | ✓ | ✓ 
Publikasi |  |  |  |  | ✓

**Comment by kahfirk** [comment]
Kahfi Rizky Kosasih (20924006)


No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi overfitting atau underfitting pada Model CNN  (Studi Kasus: Analisis Gambar Medis Pneumonia) 
1 | Data Source and Accessibility | Data diagnosis gambar medis penderita Pneumonia dari Guangzhou Women and Children's Medical Center, Guangzhou, Cina (https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia )
2 | Research Question or Focus | Memberikan gambaran visualisasi dampak dari inisialisasi kuntitas banyak filter dan neuron terhadap akurasi dan nilai dari fungsi loss yang tertinjau.
3 | Methods for Visualization | Visualisasi kontur, heatmap, dan animasi.
4 | Programming Language and Tools | Bahasa pemrograman Pythond dengan package Numpy, Matplotlib, dan Tensorflow
5 | Platforms and Technical Setup | Windows 11, Google Collab, dan Google Drive
6 | Types of Output | GIF, JPG, Jurnal atau Artikel Medium
7 | Timeline and Next Steps | (terlampir) 


Kegiatan | Minggu ke-1 | Minggu ke-2 | Minggu ke-3 | Minggu ke-4 | Minggu ke-5
:- | :- | :- | :- | :- | :-
Topik dan Data | v | - | - | - | - | - | - 
Pemodelan | v | v | - | - | - | - | - 
Visualisasi  |  - | - | v | v | - | - | - 
Penyusunan Laporan | - | - | - | v | v  
Publikasi |  - | - | - | - | v | v | v

**Comment by ariansyahedu** [comment]
Ariansyah (20924007)
No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi Temperatur di Pulau Jawa
1 | Data Source and Accessibility | Data Cuaca BMKG
2 | Research Question or Focus | Memberikan gambaran dinamika temperatur di Pulau Jawa selama tahun 2011-2023 dan perubahan-perubahannya disaat terjadi peristiwa penting.
3 | Methods for Visualization | Colored Contour Map
4 | Programming Language and Tools | Python: Numpy, Pandas, Matplotlib
5 | Platforms and Technical Setup | GitHub, VSCode, MsOffice
6 | Types of Output | Web Article
7 | Timeline and Next Steps | (Tercantum di bawah)

Kegiatan | Minggu ke- 1| Minggu ke- 2| Minggu ke- 3| Minggu ke- 4| Minggu ke- 5
:- | :- | :- | :- | :- | :-
Mencari Topik & Data | v | v | - | - | -
Data Wrangling | - | v | v | - | -
Exploratory Data Analysis | - | - | v | v | -
Visualisasi | - | - | v | v | -
membuat Laporan/ Dashboard | - | - | - | - | v

**Comment by arvincans** [comment]
Name : Arvin Cansius
NIM : 20924003

No | Key Element | Description
:- | :- | :-
0 | Tentative Title | Visualisasi dan Analisa data harga rumah123 pada bulan Oktober 2024
1 | Data Source and Accessibility | Kaggle (https://www.kaggle.com/datasets/nizarfaris/house-price-in-bandung-area)
2 | Research Question or Focus | DImana lokasi harga rumah yang paling murah?
3 | Methods for Visualization | Visualisasi peta
4 | Programming Language and Tools | Python: numpy, geopy, folium
5 | Platforms and Technical Setup | Google Colab, Github, Github.io, medium
6 | Types of Output | Medium web article, github.io
7 | Timeline and Next Steps | ...

- Week 1: Do the price data preprocessing
- Week 2 : Do the location preprocessing + visualization
- Week 3 : location visualization
- Week 4 : location visualization + setting output on github.io
- Week 5 : setting output on github.io + another media (like medium)
